org.testeditor.tcl.dsl/src/main/java/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrer.xtend

```diff
@@ -755,8 +756,8 @@ class TclJvmModelInferrer extends AbstractModelInferrer {
 	}
 
 	private def void appendReporterCall(ITreeAppendable output, SemanticUnit unit, Action action, String message, String id, Status status,
-		List<VariableReference> variables) {
-		testRunReporterGenerator.buildReporterCall(_typeReferenceBuilder?.typeRef(SemanticUnit)?.type, unit, action, message, id, status, reporterFieldName, variables,
+		List<VariableReference> variables, List<StepContentElement> amlElements) {
```

Comment:
how about overloading the method with the original signature, so that the added parameter becomes optional, and you don't have to pass in `null`?
```
private def void appendReporterCall(ITreeAppendable output, SemanticUnit unit, Action action, String message, String id, Status status, List<VariableReference> variables) {
    appendReporterCall(output, unit, action, message, id, status, variables, null);
}
```

Reply:
removed the null calls with either actual aml element lists. discarded the idea of overloading, since passing no aml elements should be a decision explicitly made for each call